src/main/java/edu/kis/vh/nursery/IStackAndListSimilarities.java

```diff
@@ -0,0 +1,14 @@
+package edu.kis.vh.nursery;
+
+public interface IStackAndListSimilarities {
```

Comment:
Names of operations do not correspond to operating the stack or list, thus do not reflect the interface name.